README.md

```diff
@@ -195,7 +195,7 @@ where `<id>` is the container ID listed in the `service ps` command.
 
 ## Production Use with Kubernetes
 
-NOTE: Please consider using the [Kubernetes Operator](https://www.hivemq.com/docs/operator/latest) instead, as it makes production deployment of HiveMQ much easier.
+NOTE: Please consider using the [Kubernetes Operator](https://docs.hivemq.com/hivemq-operator/index.html) instead, as it makes production deployment of HiveMQ much easier.
```

Comment:
```suggestion
NOTE: Please consider using the [Kubernetes Operator](https://docs.hivemq.com/hivemq-platform-operator/index.html) instead, as it makes production deployment of HiveMQ much easier.
```

Reply:
this is intentional as 4.9 is not compatible with the new platform operator